### DISCUSSION TITLE
Feature request: Slow Mo

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#15756] [Feature requests]
OS: Windows 8
Version: 0.2.5

In addition to the fast-forward time controls, it might be nice to have one or two options for slow-mo, where time is slowed down rather than sped up.

If it were up to me, I'd put in options of:

- 50% speed
- 10% speed
- Frame-by-frame mode

I'll admit it's not the kind of thing you'd use *super* often, and it should probably be available only through the drop-down menu, never automatically cycled through when hitting the fast forward button.

Potential uses I can think of:
- Making it easier to click on an elusive and fast-moving sprite, such as an individual peep in a roller coaster.
- Showcasing cool (or instructive) but short-lived moments when recording for youtube videos.
- Being better able to see who's doing what in a very crowded path.
- Being better able to catch and look at momentary glitches like #11325. 
- Being better able to watch (and perhaps figure out problems of) an extremely fast coaster or other ride.
- Being able to execute management actions and build/demolish things very fast without needing to go _all_ the way to using the build while paused cheat or use fast pause/unpause cycles. Such as when you first open a scenario that has many things wrong with it and needs to be fixed in a hurry in order to win. (This slow-mo option might have to be considered a cheat for that purpose.) Could also be useful for building a big coaster without ignoring the rest of the park for as long, but also without doing full build-while-paused mode.

**Comment by James103** [comment]
The frame-by-frame mode is already implemented in the form of a keybind, but it would be nice to have the 50% and 10% speed modes available for the purposes you mentioned without you having to repeatedly press the frame advance key or hold it, both of which can't be indefinitely sustained in a human way.

**Comment by Gymnasiast** [comment]
@James103 Please stop commenting on every single issue and PR. It's getting on our nerves.
